docs/en/observability/slo-create.asciidoc

```diff
@@ -0,0 +1,107 @@
+[[slo-create]]
+= Create a service-level objective (SLO)
+
+++++
+<titleabbrev>Create an SLO</titleabbrev>
+++++
+
+beta::[]
+
+IMPORTANT: Before creating an SLO, you need to <<slo-privileges, configure SLO access>>.
+
+To create an SLO, go to *Observability → SLOs*: 
+
+* If you're creating your first SLO, you'll see an introductory page. Click the *Create SLO* button.
+* If you have previously created SLOs, click the *Create new SLO* button in the upper-right corner of the SLO page.
+
+From here, complete the following to create an SLO:
+
+* <<define-sli,Define your service-level indicator (SLI)>>
+* <<set-slo>>
+* <<slo-describe>>
+
+[discrete]
+[[define-sli]]
+== Define your SLI
+
+You can choose one of the following SLI types:
```

Comment:
@kdelemme @CoenWarmer Is there any way we can add a why to this section, as in why the user might want to select one SLI type over the other? Maybe it's obvious and intuitive to the user and I just don't have enough experience with SLOs, but it seems like it might be useful to add.

Reply:
Does something like this work?

Depending on where the data is located, you would use one or another SLI type. For example, if you are already using APM for your services, and want to create an SLO on such service, you would use either one of the APM indicator type. If you have some raw logs coming from your services, and you want to create an SLO based on such logs, you would rely on the Custom KQL indicator type.

Reply:
Thanks, that's helpful, I'll add it in!